data/themes/default/style.css

```diff
@@ -540,28 +540,47 @@ TrackContainerView QLabel
 /* instrument pattern */
 PatternView {
 	color: rgb( 119, 199, 216 );
+	qproperty-mutedColor: rgb( 80, 80, 80 );
+	qproperty-selectedColor: rgb( 0, 125, 255 );
 	qproperty-fgColor: rgb( 187, 227, 236 );
+	qproperty-fgMutedColor: rgb( 128, 128, 128 );
 	qproperty-textColor: rgb( 255, 255, 255 );
+	qproperty-textShadowColor: rgb( 0, 0, 0 );
+	qproperty-gradient: true;
 }
 
 /* sample track pattern */
 SampleTCOView {
 	color: rgb( 74, 253, 133 );
+	qproperty-mutedColor: rgb( 80, 80, 80 );
+	qproperty-selectedColor: rgb( 0, 125, 255 );
 	qproperty-fgColor: rgb( 187, 227, 236 );
+	qproperty-fgMutedColor: rgb( 128, 128, 128 );
```

Comment:
Of no implication to you... I'm not a fan of the `fgFoo` tag, personally because normally in css `fg` is synonymous with `foreground` which is synonymous with `color` which is synonymous with `textColor`.  In this case, it's the gradient/overlay, right? Long term, I think we need to cleanup our naming conventions a bit, but you've at least kept it consistent. :+1: